proto/viam/service/motion/v1/motion.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package viam.service.motion.v1;
+package viam.service.motion.v2;
```

Comment:
since moveonglobe request has changed that is breaking, and we need to bump the version, right?

Reply:
I don't think we're adopting that convention yet although the intent is appreciated :). We just accept breaking changes for motion right now, right @raybjork and @Fahmina?

Reply:
okii, ill change back

Reply:
Yeah we did not plan to do this